doc/man3/SSL_SESSION_get0_hostname.pod

```diff
@@ -67,9 +75,9 @@ SSL_SESSION_set1_alpn_selected() functions were added in OpenSSL 1.1.1.
 
 Copyright 2016-2025 The OpenSSL Project Authors. All Rights Reserved.
 
-Licensed under the Apache License 2.0 (the "License").  You may not use
-this file except in compliance with the License.  You can obtain a copy
-in the file LICENSE in the source distribution or at
+Licensed under the Apache License 2.0 (the "License"). You may not use this
```

Comment:
Please revert this part of the change. We should keep the license text the same format.